src/controller/CommissioningDelegate.h

```diff
@@ -687,11 +687,18 @@ struct GeneralCommissioningInfo
 struct ICDManagementClusterInfo
 {
     // Whether the ICD is capable of functioning as a LIT device.  If false, the ICD can only be a SIT device.
-    bool isLIT;
+    bool isLIT = false;
     // Whether the ICD supports the check-in protocol.  LIT devices have to support it, but SIT devices
     // might or might not.
```

Comment:
This is also true for the user active mode trigger. Should we have a boolean also for this?

Reply:
for userActiveModeTriggerInstruction, https://github.com/project-chip/connectedhomeip/blob/86af23969f12ae78ffa22c4614aeddaa0f7793b8/src/controller/CHIPDeviceController.cpp#L2358

I think we don't need the boolean in struct, since we can check size of userActiveModeTriggerInstruction in other place

But for check-in protocol boolean, https://github.com/project-chip/connectedhomeip/blob/86af23969f12ae78ffa22c4614aeddaa0f7793b8/src/controller/AutoCommissioner.cpp#L755, we do have use it in other place.